Formula/s/signal-cli.rb

```diff
@@ -21,9 +21,10 @@ class SignalCli < Formula
   uses_from_macos "llvm" => :build # For `libclang`, used by `boring-sys` crate
   uses_from_macos "zip" => :build
 
-  on_linux do
-    depends_on arch: :x86_64 # `:libsignal-cli:test` failure, https://github.com/AsamK/signal-cli/issues/1787
-  end
+  # FIXME: Uncomment below when https://github.com/Homebrew/brew/issues/19838 is resolved
+  # FIXME: on_linux do
+  # FIXME:  depends_on arch: :x86_64 # `:libsignal-cli:test` failure, https://github.com/AsamK/signal-cli/issues/1787
+  # FIXME: end
```

Comment:
Or just build `libsignal-client` first and pass `gradle build -Plibsignal_client_path="path/to/libsignal-client.jar"` and arm64 Linux will work fine.

Reply:
Will try in a follow-up!